mentoring/tests/integration/test_mcq.py

```diff
@@ -231,3 +260,11 @@ def test_questionnaire_html_choices(self, page):
         self.wait_until_disabled(submit)
 
         self.assertIn('Congratulations!', messages.text)
+
+
+@mock.patch.object(MentoringBlock, 'get_theme', mock.Mock(return_value={'package': None, 'locations': []}))
```

Comment:
Do you need to have that `mock.Mock()` in the middle there? I believe you can just pass a `return_value` argument to the `patch.object` call, and it will do the right thing. I could be wrong, though.

Also, it occurs to me -- what is this `MCQBlockBlankThemeTest` class actually *for*? What does it do? It doesn't seem like there are any test cases in it.